app/views/taxons/_document_list_with_grid.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if documents.any? %>
   <div class="grid-row">
-    <div class="column-two-thirds">
+    <div class="column-two-thirds" <%= "data-module=track-click" if track_click?(documents) %>>
```

Comment:
Yes, we've added a card for it, but we didn't do the change within this piece of work as it will also mean updating all the other places where the document list is used https://trello.com/c/fDWv3Kle/117-add-track-click-wrapper-to-document-list-component